src/main/java/com/commercetools/sync/sdk2/products/utils/ProductSyncUtils.java

```diff
@@ -2,17 +2,32 @@
 
 import static com.commercetools.sync.commons.utils.CollectionUtils.emptyIfNull;
 import static com.commercetools.sync.commons.utils.OptionalUtils.filterEmptyOptionals;
-import static com.commercetools.sync.products.utils.ProductUpdateActionUtils.*;
-
+import static com.commercetools.sync.sdk2.products.utils.ProductUpdateActionUtils.buildActionIfPassesFilter;
+import static com.commercetools.sync.sdk2.products.utils.ProductUpdateActionUtils.buildActionsIfPassesFilter;
+import static com.commercetools.sync.sdk2.products.utils.ProductUpdateActionUtils.buildAddToCategoryUpdateActions;
+import static com.commercetools.sync.sdk2.products.utils.ProductUpdateActionUtils.buildChangeNameUpdateAction;
+import static com.commercetools.sync.sdk2.products.utils.ProductUpdateActionUtils.buildChangeSlugUpdateAction;
+import static com.commercetools.sync.sdk2.products.utils.ProductUpdateActionUtils.buildPublishOrUnpublishUpdateAction;
+import static com.commercetools.sync.sdk2.products.utils.ProductUpdateActionUtils.buildRemoveFromCategoryUpdateActions;
+import static com.commercetools.sync.sdk2.products.utils.ProductUpdateActionUtils.buildSetCategoryOrderHintUpdateActions;
+import static com.commercetools.sync.sdk2.products.utils.ProductUpdateActionUtils.buildSetDescriptionUpdateAction;
+import static com.commercetools.sync.sdk2.products.utils.ProductUpdateActionUtils.buildSetMetaDescriptionUpdateAction;
+import static com.commercetools.sync.sdk2.products.utils.ProductUpdateActionUtils.buildSetMetaKeywordsUpdateAction;
+import static com.commercetools.sync.sdk2.products.utils.ProductUpdateActionUtils.buildSetMetaTitleUpdateAction;
+import static com.commercetools.sync.sdk2.products.utils.ProductUpdateActionUtils.buildSetSearchKeywordsUpdateAction;
+import static com.commercetools.sync.sdk2.products.utils.ProductUpdateActionUtils.buildSetTaxCategoryUpdateAction;
+import static com.commercetools.sync.sdk2.products.utils.ProductUpdateActionUtils.buildTransitionStateUpdateAction;
+import static com.commercetools.sync.sdk2.products.utils.ProductUpdateActionUtils.buildVariantsUpdateActions;
+
+import com.commercetools.api.models.product.Product;
+import com.commercetools.api.models.product.ProductDraft;
+import com.commercetools.api.models.product.ProductProjection;
+import com.commercetools.api.models.product.ProductRemoveVariantAction;
+import com.commercetools.api.models.product.ProductUpdateAction;
 import com.commercetools.sync.products.ActionGroup;
```

Comment:
TODO?

Reply:
Modified
https://github.com/commercetools/commercetools-sync-java/commit/4feeabfcd6f91e105614fa8a38fbf136573cb6f0

Reply:
Can you check ChunkUtils.Java? 
It is still using `SphereClient`.

Reply:
Added TODO
https://github.com/commercetools/commercetools-sync-java/pull/872/commits/39c6de874b1f3ba1153ede091b96fd248e7eeae6